src/test/java/org/eclipse/yasson/defaultmapping/recursive/RecursionTest.java

```diff
@@ -0,0 +1,49 @@
+/*******************************************************************************
+ * Copyright (c) 2019 Payara Services and/or its affiliates. All rights reserved.
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License v1.0 and Eclipse Distribution License v. 1.0
+ * which accompanies this distribution.
+ * The Eclipse Public License is available at http://www.eclipse.org/legal/epl-v10.html
+ * and the Eclipse Distribution License is available at
+ * http://www.eclipse.org/org/documents/edl-v10.php.
+ *
+ * Contributors:
+ *  Payara Services - initial implementation
+ ******************************************************************************/
+package org.eclipse.yasson.defaultmapping.recursive;
+
+import static org.junit.Assert.assertEquals;
+
+import java.io.IOException;
+
+import javax.json.bind.Jsonb;
+
+import org.eclipse.yasson.internal.JsonBindingBuilder;
+import org.eclipse.yasson.serializers.model.RecursiveObject;
+import org.junit.Test;
+
+
+/**
+ * Testing for recursive object mapping
+ *
+ * @author Matt Gill
+ */
+public class RecursionTest {
```

Comment:
this test has nothing to do with custom user [de]serializers. That's fine, but I wanted to make sure it's what you intended to capture in this test?

Reply:
Yes that's expected. A test exists for custom serialisers which I modified, but this was a gap in the tests which was somewhat related to my changes.